content/docs/user-guide/overview.md

```diff
@@ -108,8 +108,8 @@ bringing best practices from software engineering into the data science field.
 \* **copy-on-write links or "reflinks"** are a relatively new way to link files
 in UNIX-style file systems. Unlike hardlinks or symlinks, they support
 transparent [copy on write](https://en.wikipedia.org/wiki/Copy-on-write). This
-means that editing a reflinked file is always safe as all the other links to the
-file will reflect the changes.
+means that editing a reflinked file is always safe as all the other linked copies
+of the file will remain unchanged.
```

Comment:
Right. So "the reflinked file" here refers to the shallow copy of file data (what you see in the workspace). Modifying that does not change the original data (in the cache), rather, a deep copy is created at that point (on-write).

Reply:
On the other hand if "the reflinked file" referred to the original cached data, I think changes to that do propagate to any shallow copies. I think that's what this was trying to say, notice wording "all the other linked copies".

Reply:
Anyhow, we should probably rewrite this completely to express the main point: reflinks seen in the workspace can be modified without any special preparation (e.g. no `dvc unprotect` needed ) and still achieving the efficiency of file links (while unmodified).

Reply:
I'm easy here, I think you know best how you want this section to be.

Jorge, to my understanding reflinks are not directional like symlinks. When you do a shallow copy, both are now the "original" and point to the same blocks on disk. Writing to either will leave the other unchanged. You can verify that quickly by using `cp -c` on MacOS or `cp --reflink=always` on Linux with a supporting filesystem.